### PR TITLE
Release apr.2020

### PR DESCRIPTION
**Pull request by AlleksD**
Update the stable version of PySIMM to be compatible with Python 3.x.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+numpy
+pandas
```
Comment (mefortunato) [inline requirements.txt]: `pytest` is missing

```suggestion
pandas
pytest

```

File: .gitpod.dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+FROM gitpod/workspace-full
```
Comment (mefortunato) [inline .gitpod.dockerfile:1]: I think gitpod is really useful. Any concerns using my personal LAMMPS docker image?

File: requirements.txt (hunk 1 above)
Comment (AlleksD) [reply requirements.txt]: actually yes, tests are working with pytest right now closely. 
Also, I will add an .md file to the test folder that briefly guides a user through the test files.

File: .gitpod.dockerfile (hunk 2 above)
Comment (AlleksD) [reply .gitpod.dockerfile:1]: gitpod looks cool indeed, totally up to that!
On your LAMMPS docker image, yes, it was blindly copied to the release. I don't have any concerns, and I remember it working well with PySIMM.
If you are fine with that, let us leave it there for this release

Comment (mefortunato) [reply .gitpod.dockerfile:1]: Sounds good to me!

**Pull request merged**